zkevm-circuits/src/evm_circuit/execution/jump.rs

```diff
@@ -80,7 +80,11 @@ impl<F: FieldExt> ExecutionGadget<F> for JumpGadget<F> {
         self.destination.assign(
             region,
             offset,
-            Some(destination.to_le_bytes()[..3].try_into().unwrap()),
+            Some(
+                destination.to_le_bytes()[..NUM_BYTES_PROGRAM_COUNTER]
```

Comment:
since we use `NUM_BYTES_PROGRAM_COUNTER` to avoid hardcode magic number here.

I think we could also do the same for https://github.com/appliedzkp/zkevm-circuits/blob/8d2eea1df59d4375901be5e901b32ee62ef06d61/zkevm-circuits/src/evm_circuit/execution/memory.rs#L167 in this PR. 

(Because it's just such a tiny modification, so I have been trying to find a PR to put it into 😂 )

Reply:
Sounds great, fixed in [`f6de529`](https://github.com/appliedzkp/zkevm-circuits/pull/236/commits/f6de529c2778489c7f6c8dcc783a8bb25be8b0fd)

Reply:
So  update ../evm_circuit/execution/pc.rs as well ?

Reply:
Which part do you mean? I think assignment from `program_counter` is already 8 bytes because it's an `u64`.

Reply:
yeah,  I am not sure if need limitation in configure constraint https://github.com/appliedzkp/zkevm-circuits/blob/main/zkevm-circuits/src/evm_circuit/execution/pc.rs#L30 , but looks trivial!